monai/bundle/scripts.py

```diff
@@ -1270,6 +1273,9 @@ def trt_export(
             that the TensorRT tries to fit. The `OPT_BATCH` should be the most frequently used input batchsize in
             the application.
         device: the target GPU index to convert and verify the model.
+        use_onnx: whether to use the ONNX-TensorRT way to export the TensorRT engine-based torchscript model.
```

Comment:
Suggest to add some docstrings in: https://github.com/Project-MONAI/MONAI/pull/6274/files#diff-a715c9207f1e727d7a6ed7c6b25dfb57045d4429fc0b4eed3f5396e26ab34f80R1246
Considering we already have `onnx_export ` and `trt_export`, users may be curious about what is "ONNX-TensorRT", is there any reference we can attach here?